scripts/templates/index.rst.mako

```diff
@@ -14,5 +14,7 @@
    core/INTRO.rst
    core/PROG.rst
    core/CONTRIB.rst
+   core/CUDA.rst
```

Comment:
Also @kbenzie I'm not sure if you want to add these to the index? If not I can remove or change the order